src/charm.py

```diff
@@ -25,24 +25,25 @@
 from ops.charm import CharmBase
 from ops.framework import StoredState
 from ops.main import main
-from ops.model import ActiveStatus, WaitingStatus
+from ops.model import ActiveStatus, RelationDataContent, WaitingStatus
+from ops.pebble import ExecProcess
 from opslib.mysql import MySQLClient
 from yaml import safe_load
 
 import exceptions
+import types_
 
 # MySQL logger prints database credentials on debug level, silence it
 logging.getLogger(mysql.connector.__name__).setLevel(logging.WARNING)
 logger = logging.getLogger()
 
 
 class WordpressCharm(CharmBase):
-    """Charm for WordPress on kubernetes."""
+    # Ignore class attributes in docstring since each methods should cover it's own.
+    """Charm for WordPress on kubernetes."""  # noqa: DCO060
```

Comment:
What's DCO060?

Reply:
It's
```
DCO060: class has one or more public attributes and the docstring does not have an attributes section.
```
but it's no longer necessary, i've added proper attributes that need to be addressed. Thank you!